modelql-core/src/commonMain/kotlin/org/modelix/modelql/core/ZipStep.kt

```diff
@@ -206,7 +207,15 @@ class ZipOutputSerializer<CommonT, Out : IZipOutput<CommonT>>(
     override val descriptor: SerialDescriptor = if (elementSerializers.size == 1) {
         elementSerializers.single().descriptor
     } else {
-        ZipNOutputDesc(elementSerializers.map { it.descriptor }.toTypedArray())
+        val typeParameters = elementSerializers.map { it.descriptor }.toTypedArray()
+        @OptIn(InternalSerializationApi::class)
```

Comment:
I reported, that we need to use this internal API for our use-case at https://github.com/Kotlin/kotlinx.serialization/issues/2899